docs/attachments/attachments-schema.yaml

```diff
@@ -15,28 +17,32 @@ definitions:
         size:
           type: number
         category:
-          type: string
-          ref: '#/definitions/resourcesCategory'
+          type: object
+          properties:
+            _id: 
+              type: string
+            name:
+              type: string
```

Comment:
Why can't you reference the schema (resourcesCategoryAttachment) that you created below?